internal/cmd/gocdk/_static/demo/runtimevar.variable/demo_runtimevar_variable.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"context"
+	"html/template"
+	"net/http"
+	"os"
+
+	"gocloud.dev/runtimevar"
+	_ "gocloud.dev/runtimevar/constantvar"
+	_ "gocloud.dev/runtimevar/filevar"
+)
+
+// TODO(rvangent): This file is user-visible, add many comments explaining
+// how it works.
+
+func init() {
+	http.HandleFunc("/demo/runtimevar.variable/", runtimevarVariableHandler)
+}
+
+var variableURL string
+var variable *runtimevar.Variable
+var variableErr error
+
+func init() {
+	variableURL = os.Getenv("RUNTIMEVAR_VARIABLE_URL")
+	if variableURL == "" {
+		variableURL = "constant://?val=my-variable&decoder=string"
```

Comment:
does this result in something functional for the demo? (It's not clear to me if this is only intended as an example of the url form or if it results in a usable thing based on `constantvar`)

Reply:
Yes, it is usable -- it ends up with a variable that always has this value.

Reply:
https://screenshot.googleplex.com/phmAJSZuikt